spec/default_spec.rb

```diff
@@ -113,7 +106,14 @@
   it do
     expect(chef_run).to install_python_package('python-openstackclient')
       .with(
-        version: '3.11.0',
+        version: '3.14.0',
+        # virtualenv: '/opt/osc'
```

Comment:
Why comment this out?

Reply:
ChefSpec doesn't support including it. See what happens when you remove the comment.